x/dex/client/cli/query_pool_reserves.go

```diff
@@ -29,16 +29,16 @@ func CmdListPoolReserves() *cobra.Command {
 
 			queryClient := types.NewQueryClient(clientCtx)
 
-			params := &types.QueryAllPoolReservesRequest{
-				PairId:  reqPairID,
-				TokenIn: reqTokenIn,
-			}
-
 			pageReq, err := client.ReadPageRequest(cmd.Flags())
```

Comment:
I'm not sure `list-pool-reserves` has pagination flags
<img width="949" alt="Screenshot 2024-03-21 at 10 58 31 PM" src="https://github.com/neutron-org/neutron/assets/6247919/d9c45300-5301-4a51-ac22-179d66fe5fae">